core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.ts

```diff
@@ -54,10 +54,11 @@ angular.module('oppia').component('skillPreviewTab', {
       var ctrl = this;
 
       ctrl.$onInit = function() {
+        ctrl.INTERATION_TYPE_ALL = 'All';
         ctrl.skillId = UrlService.getSkillIdFromUrl();
         SkillEditorStateService.loadSkill(ctrl.skillId);
         ctrl.questionTextFilter = '';
-        ctrl.interactionFilter = 'All';
+        ctrl.interactionFilter = ctrl.INTERATION_TYPE_ALL;
```

Comment:
INTERACTION is misspelled.

There are other string constants that are repeated multiple times. You need to fix those too. In addition, all usages of 'All' in this file need to be replaced with your constant.

Reply:
Fixed.